src/lib/schematics/update/tests/run-fixtures.spec.ts

```diff
@@ -0,0 +1,72 @@
+import {getSystemPath, normalize, virtualFs} from '@angular-devkit/core';
+import {TempScopedNodeJsSyncHost} from '@angular-devkit/core/node/testing';
+import {SchematicTestRunner} from '@angular-devkit/schematics/testing';
+import {readFileSync} from 'fs';
+import {join} from 'path';
+import {createTestApp, migrationCollection, runPostScheduledTasks} from '../../utils/testing';
+
+describe('update fixtures', () => {
+
+  /**
+   * Path to directories that contain a testing fixture and result fixture. These fixtures
+   * will be used to verify the update schematic.
+   */
+  const fixturesDirectories = [
+    'angular_material/src/lib/schematics/update/tests/class-names',
```

Comment:
Should this use a glob? Seems like it would be a pain to keep this up to date.

Reply:
This won't work for now because Bazel does not copy the files that are referenced in the `data` attribute of the `jasmine_node_test` rule. I'll add a TODO and revisit once we found a better solution.